tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -218,7 +237,7 @@ public void With_maxIdleTime_should_return_expected_result()
             var result = subject.With(maxIdleTime: newMaxIdleTime);
 
             result.ApplicationName.Should().Be(subject.ApplicationName);
-            result.Authenticators.Should().Equal(subject.Authenticators);
+            AssertActualAuthenticatorsFactoryResult(result.AuthenticatorFactories, subject.AuthenticatorFactories);
```

Comment:
Replace with:

            result.AuthenticatorFactories.Should().Equal(subject.AuthenticatorFactories);